pkg/cmd/cli/cli.go

```diff
@@ -0,0 +1,126 @@
+// Copyright 2021 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cli
+
+import (
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/chzyer/readline"
+	"github.com/mattn/go-shellwords"
+	"github.com/pingcap/ticdc/pkg/cmd/factory"
+	"github.com/pingcap/ticdc/pkg/cmd/util"
+	"github.com/pingcap/ticdc/pkg/logutil"
+	"github.com/spf13/cobra"
+)
+
+// options defines flags for the `cli` command.
+type options struct {
```

Comment:
I would prefer `option`

Reply:
Nice catch! Thanks. Addressed.

Reply:
Although I addressed it, I don't know if it really pays off. Because it's not uniform with other commands anymore. Also the singular and plural need to be taken care of when maintaining the code. Maybe we shouldn't obsess about singular and plural here? I don't know, what do you think? cc: @amyangfei @overvenus

Reply:
I notice kubectl uses `xxxOptions`.
Is it because there is only one option in this struct, what if we add another option? Maybe use `options` is enough. Not a big problem